test/Microsoft.IdentityModel.TestUtils/Default.cs

```diff
@@ -96,6 +96,10 @@ public static SigningCredentials AsymmetricSigningCredentials
         {
             get => new SigningCredentials(KeyingMaterial.DefaultX509SigningCreds_2048_RsaSha2_Sha2.Key, KeyingMaterial.DefaultX509SigningCreds_2048_RsaSha2_Sha2.Algorithm, KeyingMaterial.DefaultX509SigningCreds_2048_RsaSha2_Sha2.Digest);
         }
+        public static X509SigningCredentials X509AsymmetricSigningCredentials
```

Comment:
nit: can you add a new line after the closing brace?

Reply:
done.